doc/translations/fr.po

```diff
@@ -35852,7 +35852,7 @@ msgstr ""
 
 #: doc/classes/PhysicsServer2D.xml:888
 msgid ""
-"Sets a damped spring joint parameter. See [enum DampedStringParam] for a "
+"Sets a damped spring joint parameter. See [enum DampedSringParam] for a "
```

Comment:
You lost the "p" here. But no need to update the .po and .pot files, this is done via a script once in a while.

Reply:
Fixed it anyway. ;-)